classes/services/PKPSubmissionService.inc.php

```diff
@@ -317,7 +317,7 @@ public function getPropertyReviewAssignments($submission) {
 
 			$request = \Application::get()->getRequest();
 			$currentUser = $request->getUser();
-			$context = $request->getContext();
+			$context = Services::get('context')->get($submission->getData('contextId'));
```

Comment:
Sorry to keep sending you back on this issue -- but this will potentially add a lot of overhead, as it'll mean fetching the object repeatedly if called on a  list of submissions. I would suggest using the same optimization you see elsewhere:
```php
if (!$context || $context->getId() != $submission->getData('contextId')) {
    $context = Services::get('context')->get($submission->getData('contextId'));
}
```
This will probably be faster, but still safe for batch jobs.